safe/gui/tools/wizard/step_kw33_multi_classifications.py

```diff
@@ -103,6 +103,7 @@ def __init__(self, parent=None):
 
         # GUI, good for testing
         self.save_button = None
+        self.restore_defaukt_button = None
```

Comment:
s/defaukt/default/g

It's everywhere in your code

Reply:
nooo, typo while doing refactoring :D, thanks